digits/dataset/generic/test_views.py

```diff
@@ -1,13 +1,16 @@
 # Copyright (c) 2016, NVIDIA CORPORATION.  All rights reserved.
 from __future__ import absolute_import
 
-import json
-
 from bs4 import BeautifulSoup
+import json
+import numpy as np
```

Comment:
I think what they're distinguishing between are packages that come with a standard Python install (i.e. `apt-get install python`) vs. add-on packages (i.e. `pip install Flask`). At least, that's been my interpretation. I'm open to push-back if you think it's dumb.
```py
import json
import os
import tempfile

from bs4 import BeautifulSoup
import numpy as np
import PIL.Image

from digits import extensions
import digits.test_views
from digits.utils import constants
```